SkeletonViewCore/Sources/API/Appearance/SkeletonAppearance.swift

```diff
@@ -37,5 +37,7 @@ public class SkeletonViewAppearance {
 
     public var renderSingleLineAsView: Bool = false
     
+    public var skeletonCornerRadius: Float = 0
```

Comment:
@Juanpe Should I use `CGFloat` in here?

Reply:
Well, we should consider it, but this would be a breaking change because this is a `public` property and I don't have planned to release a major version soon. So, we could leave it as `Float` for now.